var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -184,6 +189,9 @@ def cmake_args(self):
         with when('+openpmd'):
             args.append('-DWarpX_openpmd_internal=OFF')
 
+        if '+sensei' in spec:
+            args.append(self.define('SENSEI_DIR', os.path.join(spec['sensei'].prefix.lib, 'cmake')))
```

Comment:
Please use Spack's `join_path` instead of `os.path` (please see examples in other locations of the same file :) )